MODULE.bazel

```diff
@@ -0,0 +1,46 @@
+JDK_VERSIONS = ["11", "15", "16", "17"]
+PLATFORMS = ["linux", "macos", "macos_aarch64", "win"]
+
+# Remote JDK repos for those Linux platforms are only defined for JDK 11.
+EXTRA_REMOTE_JDK11_REPOS = [
+    "remotejdk11_linux_aarch64",
+    "remotejdk11_linux_ppc64le",
+    "remotejdk11_linux_s390x",
+]
+
+REMOTE_JDK_REPOS = [("remotejdk" + version + "_" + platform) for version in JDK_VERSIONS for platform in PLATFORMS] + EXTRA_REMOTE_JDK11_REPOS
+
+REMOTE_JAVA_TOOLCHAINS = [("@" + name + "_toolchain_config_repo//:toolchain") for name in REMOTE_JDK_REPOS]
+
+module(
+    name = "rules_java",
+    version = "4.0.0",
+    compatibility_level = 1,
+    toolchains_to_register = [
+        "//toolchains:all",
+        "@local_jdk//:runtime_toolchain_definition",
+    ] + REMOTE_JAVA_TOOLCHAINS,
+)
+
+bazel_dep(name = "bazel_skylib", version = "1.0.3")
```

Comment:
It looks like skylib is not used in rules_java. Perhaps keep it simple and remove it.

Reply:
Done